src/bundle/Resources/public/scss/_main-row.scss

```diff
@@ -0,0 +1,68 @@
+.ez-content-view,
```

Comment:
In this solution, `ez-main-row` has no styles by default. Only by parents. I would suggest do something like that:
```scss
.ez-main-row {
    display: grid;
    grid-template-columns: 1fr calculateRem(80px);
    grid-template-rows: auto;
    grid-template-areas: "ez-content-container ez-context-menu";

    .ez-content-container {
        grid-area: ez-content-container;
    }

    .ez-context-menu {
        grid-area: ez-context-menu;
    }
}

.ez-content-view,
.ez-bookmark-list-view,
.ez-trash-list-view,
.ez-drafts-list-view {
    .ez-main-row {
        grid-template-columns: calculateRem(80px) 1fr calculateRem(80px);
        grid-template-areas: "ez-side-menu ez-content-container ez-context-menu";

        .ez-side-menu {
            grid-area: ez-side-menu;
        }
    }
}
```

Thanks to this `ez-main-row` has styles (the default view is when there is only one sidebar) and when needed is modified by parent.
Also you do not have to add that many `body_class` parents.